### PR TITLE
[Merged by Bors] - feat(Condensed): explicit sheaf condition for light condensed sets

### PR DESCRIPTION
**Pull request by dagurtomas**
- [x] depends on: #13501 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Condensed/Light/Explicit.lean [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+variable {A : Type*} [Category A]
+
+namespace LightCondensed
+
+/--
+The light condensed object associated to a presheaf on `LightProfinite` which preserves finite
+products and satisfies the equalizer condition.
+-/
+noncomputable def ofSheafLightProfinite (F : LightProfinite.{u}ᵒᵖ ⥤ A) [PreservesFiniteProducts F]
```
Comment (joelriou) [inline Mathlib/Condensed/Light/Explicit.lean:35]:
```suggestion
@[simps]
noncomputable def ofSheafLightProfinite (F : LightProfinite.{u}ᵒᵖ ⥤ A) [PreservesFiniteProducts F]
```

File: Mathlib/Condensed/Light/Explicit.lean [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    val := F
+    cond := by
+      rw [isSheaf_iff_preservesFiniteProducts_and_equalizerCondition F]
+      exact ⟨⟨⟨fun _ _ ↦ inferInstance⟩⟩, hF⟩
+
+/--
+The light condensed object associated to a presheaf on `LightProfinite` whose postcomposition with
+the forgetful functor preserves finite products and satisfies the equalizer condition.
+-/
+noncomputable def ofSheafForgetLightProfinite
```
Comment (joelriou) [inline Mathlib/Condensed/Light/Explicit.lean:46]:
```suggestion
@[simps]
noncomputable def ofSheafForgetLightProfinite
```

**Comment by joelriou** [comment]
Thanks!

bors d+

**Comment by dagurtomas** [comment]
bors merge

**Pull request closed without merging**